src/net.cpp

```diff
@@ -1973,8 +1986,9 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
         }
 
         if (addrConnect.IsValid()) {
-
-            if (fFeeler) {
+            if (anchor) {
```

Comment:
> Why are these specialized logs required?

Because anchors are specialized connections by their nature. As a secure feature I'd keep logs as they are.

Reply:
I really don't like this specialized logging for anchor connections (or the feeler connections below). All connection types are specialized in some way? Why not just log unconditionally for all connection types?

Reply:
> Why not just log unconditionally for all connection types?

It could be very noisy in logs. Anyway, touching logging for all connections seems out of this PR scope.

Reply:
oh, I didn't realise that these are unconditional logs. I definitely think they should be in the NET category.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-690307345).

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-691508148).